patches/chrome-browser-ui-views-passwords-password_save_update_view.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/views/passwords/password_save_update_view.cc b/chrome/browser/ui/views/passwords/password_save_update_view.cc
+index 9c02474ee0da521dcc35d36b957b6097ce6204d8..42b168292097039e08affdcf25c72588cc3ff4aa 100644
+--- a/chrome/browser/ui/views/passwords/password_save_update_view.cc
++++ b/chrome/browser/ui/views/passwords/password_save_update_view.cc
+@@ -416,6 +416,7 @@ void PasswordSaveUpdateView::AddedToWidget() {
+ 
+ void PasswordSaveUpdateView::OnThemeChanged() {
+   PasswordBubbleViewBase::OnThemeChanged();
++  BRAVE_PASSWORD_SAVE_UPDATE_VIEW_ON_THEME_CHANGED
```

Comment:
A patch instead of an override of PasswordSaveUpdateView::OnThemeChanged because PasswordSaveUpdateView's destructor is private.